docs/root/intro/version_history.rst

```diff
@@ -13,6 +14,7 @@ Version history
 * tracing: when using the zipkin tracer, it is no longer necessary to propagate the
   :ref:`x-ot-span-context <config_http_conn_man_headers_x-ot-span-context>` header.
   See more on trace context propagation :ref:`here <arch_overview_tracing>`.
+* listeners: added :ref:`transparent <envoy_api_field_Listener.transparent>` option.
```

Comment:
nit: in the future please alpha order the functional area ("listeners"). Don't worry about it for now though. I will fix in the larger release note merge I am doing.